xbmc/windowing/android/WinSystemAndroidGLESContext.cpp

```diff
@@ -7,12 +7,14 @@
  */
 
 #include "WinSystemAndroidGLESContext.h"
-
 #include "VideoSyncAndroid.h"
+
```

Comment:
remove the empty line, https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#7-headers